features/reviews/reviews.feature

```diff
@@ -0,0 +1,86 @@
+@reviews
+Feature: Reviews
+    In orders to know customer's opinions about my products
+    As a store owner
+    I want to be able to manage reviews
+
+    Background:
+        Given there are following users:
+            | email          | enabled  | created at          |
+            | beth@foo.com   | yes      | 2010-01-02 12:00:00 |
+            | martha@foo.com | yes      | 2010-01-02 13:00:00 |
+            | rick@foo.com   | yes      | 2010-01-03 12:00:00 |
+        And there are following taxonomies defined:
+            | name     |
+            | Category |
+        And taxonomy "Category" has following taxons:
+            | Clothing > T-Shirts     |
+            | Clothing > PHP T-Shirts |
+            | Clothing > Gloves       |
+        And the following products exist:
+            | name             | price | taxons       | average_rating |
+            | Super T-Shirt    | 19.99 | T-Shirts     | 0              |
+            | Black T-Shirt    | 18.99 | T-Shirts     | 0              |
+            | Sylius Tee       | 12.99 | PHP T-Shirts | 0              |
+            | Symfony T-Shirt  | 15.00 | PHP T-Shirts | 5              |
```

Comment:
Why it's avergae rating is 5 if there's only one review with lowest rating possible?